test_requirements.txt

```diff
@@ -1,7 +1,7 @@
 coveralls
```

Comment:
Should the pysat version limit in `requirements.txt` also be updated?

Reply:
I don't see a changed requirement.  Is this just to force the bug fixes in the new one?

Reply:
There isn't a changed requirement, this would be a new requirement since some of the changes here use the newer keywords, et cetera.